plugins/wasm-rust/src/log.rs

```diff
@@ -68,4 +69,45 @@ impl Log {
     pub fn critical(&self, msg: &str) {
         self.log(LogLevel::Critical, msg)
     }
+
```

Comment:
examples can be added in the comments, such as log.debugf (format_args! ("Hello, {}!", "world");